### DISCUSSION TITLE
Is there a version that works for M1 Macbook? If not are there any similar alternatives?

### DISCUSSION THREAD

**Discussion by zcecwib** [Tyrrrz/YoutubeDownloader#305] [Q&A]
Can't find one that is easy to install like this one...

**Comment by Tyrrrz** [comment]
No, but maybe you can try running it via Bootcamp (don't have a Mac myself to try). Otherwise, there are probably other YouTube apps for Mac.

**Comment by exusxt** [comment]
You need to buy parallels and install windows 11 arm then you can use the app, no other options on m1 macbook as bootcamp is not supported on arm.

**Comment by charlottepercy** [comment]
there were M1-based MacBook Air and MacBook Pro models available. If you're looking for a similar alternative to the **[MacBook 12"](https://globaltimeusa.com/macbook-12in-m7/)** with M7, you can consider the M1 MacBook Air or M1 MacBook Pro, which offer powerful performance, energy efficiency, and a compact form factor. Keep in mind that my information might be outdated, so it's best to check the latest product offerings from Apple to see if there have been any updates or new releases.
